Formula/op25.rb

```diff
@@ -0,0 +1,64 @@
+class Op25 < Formula
+  desc "Software-defined analyzer for APCO P25 signals"
+  homepage "https://osmocom.org/projects/op25/wiki"
+  url "git://op25.osmocom.org/op25.git"
+
+  depends_on "cmake" => :build
+  depends_on "cppunit" => :build
+  depends_on "swig" => :build
+  depends_on "gnuradio"
```

Comment:
I believe this defaults to building with macOS system Python 2.7 but I think should also build successfully with Python 3 (refs https://github.com/Homebrew/homebrew-core/pull/54848). I tested building against the `gnuradio@3.8` formula from https://github.com/Homebrew/homebrew-core/pull/49733 and it appeared to work.